src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -46,6 +55,41 @@ def get(self, request, organization):
             status__in=[RuleStatus.ACTIVE, RuleStatus.INACTIVE], project__in=project_ids
         )
 
+        teams = request.GET.getlist("team", [])
+        if teams:
+            unassigned = None
+            if "unassigned" in teams:
+                teams.remove("unassigned")
+                unassigned = Q(owner_id=None)
+
+            if "myteams" in teams:
```

Comment:
Are these values visible to the user at all?

Reply:
Right, I meant are we specifically using this in the ui. Sounds like we're not, just as a query param

Reply:
Aah right. Yeah. I thought you might have been asking because of the perspective ("my" is only relevant from the users perspective...thought for a good minute about whether this should be something like "theirteams" :trollface: and more seriously about other alternatives)